<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>play_double</class>
 <widget class="QMainWindow" name="play_double">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1256</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1256</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>钩子英雄</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="8,2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/game/img/G_BG_1.png);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget{
	background-color: qlineargradient(spread:reflect, x1:0.480198, y1:0.517, x2:0.480198, y2:0, stop:0.613861 rgba(6, 1, 66, 255), stop:1 rgba(141, 130, 75, 245));
	
}
#widget *{}

</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1,5,2,4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 20pt &quot;Forte&quot;;
background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="text">
            <string>   Gold Left   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_5">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Forte&quot;;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.802, fx:0.5, fy:0.5, stop:0 rgba(255, 251, 193, 255), stop:1 rgba(238, 204, 101, 255));
</string>
         </property>
         <property name="text">
          <string>goal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_6">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">border-image: url(:/game/img/LOL.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Forte&quot;;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.802, fx:0.5, fy:0.5, stop:0 rgba(255, 251, 193, 255), stop:1 rgba(238, 204, 101, 255));</string>
           </property>
           <property name="text">
            <string>tools</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,1,2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true">border-image: url(:/game/img/item_3.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_7">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);font: 12pt &quot;Forte&quot;;</string>
               </property>
               <property name="text">
                <string>ex power</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="tool1">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,1,2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">border-image: url(:/game/img/item_4.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);font: 12pt &quot;Forte&quot;;</string>
               </property>
               <property name="text">
                <string>ex money</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="tool2">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="5,1,2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="styleSheet">
                <string notr="true">border-image: url(:/game/img/item_1.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);font: 12pt &quot;Forte&quot;;</string>
               </property>
               <property name="text">
                <string>ex speed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="tool3">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="5,1,2">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">border-image: url(:/game/img/item_2.png);</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_10">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 127);font: 12pt &quot;Forte&quot;;</string>
               </property>
               <property name="text">
                <string>  Boom!  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="tool4">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLCDNumber" name="timeleft">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_11">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);font: 20pt &quot;Forte&quot;;</string>
           </property>
           <property name="text">
            <string>      timeleft       </string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
